it/src/main/java/org/corfudb/universe/node/server/SupportServerParams.java

```diff
@@ -0,0 +1,73 @@
+package org.corfudb.universe.node.server;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Builder.Default;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.ToString;
+import org.corfudb.universe.node.Node.NodeParams;
+import org.corfudb.universe.node.Node.NodeType;
+import org.corfudb.universe.node.server.CorfuServer.Mode;
+import org.corfudb.universe.node.server.CorfuServer.Persistence;
+import org.slf4j.event.Level;
+
+import java.time.Duration;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+@Builder
+@AllArgsConstructor
+@EqualsAndHashCode(exclude = {"logLevel", "stopTimeout"})
+@ToString
+public class SupportServerParams implements NodeParams {
+    private static final Map<NodeType, Integer> ports = ImmutableMap.<NodeType, Integer>builder()
+            .put(NodeType.METRICS_SERVER, 9090)
+            .build();
+
+    private final int METRICS_INTERFACE = 9090;
+
+    @Default
+    @NonNull
+    @Getter
+    private final Mode mode = Mode.CLUSTER;
+
+    @Default
+    @NonNull
+    @Getter
+    private final Persistence persistence = Persistence.DISK;
+
+    @Default
+    @NonNull
+    @Getter
+    private final Level logLevel = Level.DEBUG;
+
+    @NonNull
+    @Getter
+    private final NodeType nodeType;
+
+    @Getter
+    @NonNull
+    private final String clusterName;
+
+    @Getter
+    @Default
+    @NonNull
+    private final Duration stopTimeout = Duration.ofSeconds(1);
+
+    @Override
+    public String getName() {
+        return clusterName + "-support-node-" + getNodeType();
+    }
+
+    @Override
+    public Set<Integer> getPorts() {
+        return Optional.of(ImmutableSet.of(ports.get(getNodeType())))
```

Comment:
don't need optional, just use ImmutableSet.of(ports.get(getNodeType()) should work

Reply:
Done.